### PR TITLE
Allow excluding wildcards in paths

### PR DESCRIPTION
**Pull request by neilkakkar**
## Changes

Wildcards can now be used in exclude events filter

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests
- [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious
- [ ] New/changed UI is decent on smartphones (viewport width around 360px)

### CODE REVIEW

**Review by paolodamico — COMMENTED** [review]
Conceptually this makes sense to me, but I'm not the right person to review this, I'm not sure what the notation `2_/3` means, can you clarify? Maybe it's better just to wait for @EDsCODE

File: ee/clickhouse/queries/test/test_paths.py [hunk 1]
```diff
@@ -915,6 +915,95 @@ def test_event_inclusion_exclusion_filters(self):
...
+            timestamp="2012-01-01T03:24:34.000Z",
+        )
+
+        filter = PathFilter(
+            data={
+                "step_limit": 4,
+                "date_from": "2012-01-01",
+                "exclude_events": ["/bar/*/foo"],
+                "include_event_types": ["$pageview"],
+                "path_groupings": ["/bar/*/foo"],
```
Comment (EDsCODE) [inline ee/clickhouse/queries/test/test_paths.py:998]: maybe worth adding more tests cases such as excluding events that aren't grouped?

Comment (neilkakkar) [reply ee/clickhouse/queries/test/test_paths.py:998]: Those already exist: https://github.com/PostHog/posthog/blob/master/ee/clickhouse/queries/test/test_paths.py#L774

Comment (EDsCODE) [reply ee/clickhouse/queries/test/test_paths.py:998]: Oh i meant if you specified exclusion groupings but didn't have the corresponding path grouping

Comment (neilkakkar) [reply ee/clickhouse/queries/test/test_paths.py:998]: Ah makes sense

**Pull request merged**